server/src/db/repositories/recipe.repository.ts

```diff
@@ -0,0 +1,47 @@
+import { ErrorUtils } from "./../../logic/utils/index";
+import { DbError, ERRORS, Recipe } from "./../../models/index";
+
+export class RecipeRepository {
+
+  private _GET_BY_ID: string = `select * 
+                                from recipes
+                                where recipes.id = $1`;
+
+  private _SAVE: string = `insert into recipes(
+                             title, userId, description,
+                             imageUrl, created)
+                           values($1, $2, $3, $4, $5)
+                           returning id`;
+
+  constructor (private _db: any) {}
```

Comment:
Oh Oh, I never had seen these variable declarations since they are placed under the constructor. So I thought only the any type was used. That happens if I try to do this as fast as possible in the evening. Sorry.